### ISSUE TITLE
2018.02.08 Growth call

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/growth#37]
- **Date:** Thu Feb 8st 
 - **Time:** 11:15a–12:15p Pacific / 8:15p–9:15p Central European
 - **Place:** YouTube livestream, see Call Details below

If you want to help Bisq grow, then this is the meeting to attend!

Everyone is welcome, and there's no need to RSVP or register, but if you do plan to attend, **please add a 👍 below** so we have an idea how many people will be there.

Note also that you can subscribe to the Bisq DAO Google Calendar to get notifications about this call and other Bisq events. See https://bisq.network/calendar.

## Agenda

You'll find the general agenda of our growth calls here: https://github.com/bisq-network/growth/blob/master/docs/growth-meeting.md

As decided on our first growth call (6th of Dec '17) we will focus all our efforts for the next two months on increasing liquidity on the Bisq network.

Before we can grow the number of overall traders on the Bisq network, we first need to make sure that we have plenty of liquidity available in major trading pairs, esp. EUR and USD.
 
This means that anyone who considers themselves a potential 'market maker' or 'liquidity provider' is especially welcome on this call, to help us brainstorm ways to make Bisq an attractive place to be for you. We already have a number of ideas, and are looking to get your feedback.
 
Note that while we're happy to have pro and semi-pro traders serve as liquidity providers, we're also looking for anyone who simply has a need to sell BTC on a regular basis, e.g. for basic cash flow purposes.

**This week will be about:**
- Payment methods for US
- Define actions to promote USD/BTC liquidity week

Feel free to add any remarks below or points you want to discuss on this weeks growth call! 

## Call details
We are streaming the meeting live on YouTube this time:
- https://www.youtube.com/watch?v=pkTPUt1j9Hg

You can just watch the live stream and add comments in the live chat or if you want you can join the call by using following Google Hangout invitation link:
https://hangouts.google.com/hangouts/_/jznprr63jnapbmnjbm6uaxk65ye

## Comments welcome

Please add comments below with any questions you have or regarding any other topics you'd like to discuss on this call. See also the open issues in this repository.

Hope to see you there!

**Comment by flix1** [comment]
Now that Bisq USD Liquidity Week has a firm date and we have 5 potential participants who have shown interest in slack (**jrwashburn**, **Bsqman**, **Stephen Gornick**, **cadayton**, **manolo**) this has all the ingredients to be a vey valuable experiment. We will try to convince them to join the call.

**Comment by ManfredKarrer** [comment]
@flix1 National Bank transfer or Transfer with the same bank have the problem that ACH can be used and ACH has high chargeback risks. I am not aware of a method to enforce Wire and prevent people to use ACH for those. That was the reason why we deactivated national bank transfers in the US.

**Comment by cadayton** [comment]
Hi All, Watched the youtube video on today's meeting.   So far Zella/Chase Quickpay has been working very well for me.  For Zella through Chase, I have 2k limit per day.  I've hit this limit several times but not a big deal because I can just delay the payment till the next day.  Several times, I've received a SMS message from Chase wanting to verify, I made the transaction.  The main reason for using Zella is because that appears to be most used option in the US.  I'm open to using other payment methods and helping out with any testing that might be desired.  I run both a MacOS (LTC base) and a Windows 10 (BTC base) Bisq software.  I use the Bisq wallet for my trading pot and use the Exodus wallet for my HODL pot.  I written my own version of a hardware wallet using open sourced software using PowerShell to protect my HODL pot.  May be of interest to some.  https://github.com/cadayton/Moolah.

**Comment by cadayton** [comment]
It would be nice to have an additional filter on the buy/sell page to only display offers within a specific percentage range of the current market rate.  This is not an issue now because there are not the many offers on the table.  But if there were pages of offers, I'd like to only display offers within 3% of the market.

The one other issue I've had.  Is I made a sell offer with LTC using Zella and quickly realized that the volume of trades with LTC was extremely low and the offer would likely be there for weeks.  There should be some sort of timeout value for offers not taken where there isn't a trading fee penalty.  I've not been back to trade LTC since then.

I'm really start to fall in love with Bisq.  Excellent Excellent job you all.

**Comment by csacher** [comment]
Notes of the call: https://docs.google.com/document/d/1nsHLYddgdxm7GDlgMqR6252SQGOWjCs7-q_KyDe4FiU/edit?usp=sharing
